doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -81,62 +81,63 @@ The Kubernetes Node OS (as with any OS) consists of two main components:
+abstracted by the Container Host OS, based on API instructions from the kubelet.
+
+There are a number of different container runtimes. The simplest form of runtimes,
+the low-level container runtimes, only manage the operating system capabilities,
+such as cgroups and namespaces, and then run commands from within those cgroups
+and namespaces. An example of this type of runtime is runc, which underpins many
+of the higher-level runtimes and is considered a reference implementation of the
+`Open
 Container Initiative (OCI) runtime
-spec <https://github.com/opencontainers/runtime-spec>`__. This specification
-includes details on how an implementation (i.e. an actual container runtime such
-as runc) must, for example, configure resource shares and limits (e.g., CPU,
-Memory, IOPS) for the containers that Kubernetes (via the kubelet) schedules on
-that host. This is important to ensure that the features and capabilities
-described in :doc:`ref_model:chapters/chapter05` are
-supported by this RA and delivered by any downstream Reference Implementations
+spec <https://github.com/opencontainers/runtime-spec>`__. This specification includes
+details on how an implementation (that is, an actual container runtime such as runc)
+must, for example, configure resource shares and limits (such as CPU, Memory, IOPS)
+for the containers that Kubernetes (via the kubelet) schedules on that host. This is
+important to ensure that the features and capabilities described in :doc:`ref_model:chapters/chapter05`
+are supported by this RA and delivered by any downstream Reference Implementations
 (RIs) to the instance types defined in the RM.
 
-Where low-level runtimes are used for the execution of a container within an OS,
-the more complex/complete high-level container runtimes are used for the general
-management of container images - moving them to where they need to be executed,
-unpacking them, and then passing them to the low-level runtime, which then
-executes the container. These high-level runtimes also include a comprehensive
-API that other applications (e.g., Kubernetes) can use to interact and manage the
-containers. An example of this type of runtime is containerd, which provides the
-features described above, before passing off the unpacked container image to
-runc for execution.
+Where low-level runtimes are used for the execution of a container within an operating
+system, the more complex and complete high-level container runtimes are used for the
+general management of container images - moving them to where they need to be executed,
+unpacking them, and then passing them to the low-level runtime, which then executes the
+container. These high-level runtimes also include a comprehensive API that other
+applications, such as Kubernetes, can use to interact and manage the containers. An
+example of this type of runtime is containerd, which provides the features described
+above, before passing off the unpacked container image to runc for execution.
```

Comment:
The sentence is awkward:
An example of this type of runtime is containerd, which provides the features described above, before passing off the unpacked container image to runc for execution.

the features above include, "unpacking them, and then passing them to the low-level runtime"

Reply:
> The sentence is awkward: An example of this type of runtime is containerd, which provides the features described above, before passing off the unpacked container image to runc for execution.
> 
> the features above include, "unpacking them, and then passing them to the low-level runtime"

Fixed in [8eeb6e9](https://github.com/cntt-n/CNTT/pull/3217/commits/8eeb6e9c166cc3a4166238034dc17583f2e5fbac)